marshaling_test.go

```diff
@@ -533,11 +534,12 @@ func (s *Zuite) TestStructScan_refsNestedMoreThanOnce() {
 	}
 
 	type level1 struct {
-		L2 level2 `ws:"ws"`
+		L2 *level2 `ws:"ws"`
```

Comment:
sounds good. had changed it since there's a level3 non-pointer being tested and didn't want to modify the all_types schema to add a second ws field.

changed it to rely on multi-mapping of ws->struct